proto/data.proto

```diff
@@ -252,6 +252,22 @@ message Transaction {
   optional NodeList certain_nodes = 12 [(gogoproto.nullable) = false];
 }
 
+// Lease contains information about leader leases including the
+// expiration and lease holder.
+message Lease {
+  // The expiration is in unix nanos and is set according to the wall
+  // clock at the lease requestor / grantee.
+  optional int64 expiration = 1 [(gogoproto.nullable) = false];
+  // The leadership term for this lease.
+  optional uint64 term = 2 [(gogoproto.nullable) = false];
+  // The node ID of replica holding the lease.
```

Comment:
Why don't we send a Replica object instead to eliminate confusion? I dislike the idea of dropping information here which is part of identifying a replica, even if it's not strictly necessary at this point.